sqlmesh/core/config/connection.py

```diff
@@ -35,6 +35,7 @@
 
 class ConnectionConfig(abc.ABC, BaseConfig):
     concurrent_tasks: int
+    register_comments: bool = False
```

Comment:
I don't think it's enough to just add it here. `ConnectionConfig` is an abstract base class and not a pydantic model. I believe you need to include it explicitly in all child classes. See how `concurrent_tasks` is used.

At the same time my recollection might be inaccurate I encourage you to test my claim.

Reply:
Also I thought we wanted it to be `True` for everything but Snowflake.